### PR TITLE
Update for JSKit 0.11.1, add async `task` modifier

### PR DESCRIPTION
**Pull request by MaxDesiatov**
I've also moved sources in `TokamakDemo` directory into their respective subdirectories for easier navigation

### CODE REVIEW

**Comment by MaxDesiatov** [comment]
This is finally ready for review now, sorry for excessive notifications pinging with failed builds.

File: Sources/TokamakDemo/Modifiers/ShadowDemo.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TokamakShim
+
+struct ShadowDemo: View {
```
Comment (MaxDesiatov) [inline Sources/TokamakDemo/Modifiers/ShadowDemo.swift:17]: This was moved from existing `ShadowDemo.swift`, but for some reason git didn't pick it up.

**Review by yonihemi — APPROVED** [review]

File: Package.swift [hunk 2]
```diff
@@ -4,6 +4,31 @@
...
+  ),
+  .product(
+    name: "JavaScriptKit",
+    package: "JavaScriptKit",
+    condition: .when(platforms: [.wasi])
+  ),
+  "OpenCombineJS",
+]
+
+#if compiler(>=5.5) && canImport(Concurrency)
```
Comment (ezraberch) [inline Package.swift:22]: This conditional fails for me (Monterey/Xcode 13.1), while the following works.

```suggestion
#if compiler(>=5.5) && canImport(_Concurrency)
```

Comment (MaxDesiatov) [reply Package.swift]: I've replaced it with `#if compiler(>=5.5) && (canImport(Concurrency) || canImport(_Concurrency))`, which hopefully covers all possible cases. Does that fix the issue on your side?

Comment (ezraberch) [reply Package.swift]: Yes, that works.

File: README.md [hunk 3]
```diff
@@ -113,20 +113,33 @@ app.
 
 ## Requirements for app developers
 
-- macOS 11 and Xcode 13.0. *Xcode 13.1 is currently not supported.*
+- macOS 11 and Xcode 13.0 or later. Xcode 13.2 or later are recommended if you're developing
```
Comment (ezraberch) [inline README.md:116]:
```suggestion
- macOS 11 and Xcode 13.0 or later. Xcode 13.2 or later is recommended if you're developing
```

**Review by ezraberch — APPROVED** [review]

**Pull request merged**